### DISCUSSION TITLE
FYI: trying to add support in jupytext

### DISCUSSION THREAD

**Discussion by HaveF** [janpfeifer/gonb#120] [General]
https://github.com/mwouts/jupytext/issues/1244

**Comment by janpfeifer** [comment]
This is super cool @HaveF .

Let me know if you find any road-blockers you would need some help with.

**Reply by HaveF** [reply]
Hi, @janpfeifer jupytext's author is helping. We need some advice. Can you give some good tastes about corresponding notebook and jupytext problem?
https://github.com/mwouts/jupytext/issues/1244

**Reply by janpfeifer** [reply]
Thanks for doing this @HaveF !!

**Comment by janpfeifer** [comment]
With the new release of Jupytext, I also just [released v0.10.2](https://github.com/janpfeifer/gonb/releases/tag/v0.10.2) and updated the [gonb_jupyterlab docker](https://hub.docker.com/r/janpfeifer/gonb_jupyterlab).

Closing the discussion for now, but let me know if you or anyone finds some issue.

thanks again @HaveF !

**Reply by HaveF** [reply]
@janpfeifer It's my pleasure 🎉🎉🎉
